AutoDraft/NHL_API.py

```diff
@@ -0,0 +1,499 @@
+                                                                                                                    results.loc[player_name, 'train'+metric],
+                                                                                                                    results.loc[player_name, 'test'+metric]), # TODO: change to MASE
+                            plot_height=300, plot_width=800, tools="xpan", toolbar_location='above',
+                            x_axis_type="datetime", x_axis_location="below", x_range=(dates[0], dates[-1]),
+                            background_fill_color="#efefef")
+
+    hover_tool = HoverTool(tooltips=[("date", "@date"),
+                                    ("points", "@points")],
+                            mode='vline')
+
+    player_line.circle('date', 'points', source=real_source, line_color='blue', legend='actual')
+    player_line.line('date', 'points', source=pred_source, line_color='red', legend='predicted')
+    # player_line.patch(x=dates[-intervals.shape[0]:], y=intervals, line_color='red', alpha=0.4)
+
+    player_line.varea(x=interval_dates[:, 0], y1=intervals.loc[:, 'high'], y2=intervals.loc[:, 'low'], fill_alpha=0.4, color='red', legend='predicted')
+    # interval_glyph = Patch(x='date', y='points', fill_color="red", fill_alpha=0.4)
+    # player_line.add_glyph(interval_source, interval_glyph)
+
+    player_line.legend.location = 'top_left'
+    player_line.legend.click_policy = 'hide'
+    player_line.add_tools(hover_tool)
+    player_line.toolbar.active_multi = hover_tool
+    # player_line.yaxis.axis_label('Cumulative Points')
+
+    test_start = Span(location=start_date,
+                              dimension='height', line_color='green',
+                              line_dash='dashed', line_width=3)
+    player_line.add_layout(test_start)
+
+    select = figure(title="Drag the middle and edges of the selection box to change the range above",
+                    plot_height=130, plot_width=800, y_range=player_line.y_range,
+                    x_axis_type="datetime", y_axis_type=None,
+                    tools="", toolbar_location=None, background_fill_color="#efefef", x_range=(dates[0], dates[-1]))
+
+    range_tool = RangeTool(x_range=player_line.x_range)
+    range_tool.overlay.fill_color = "navy"
+    range_tool.overlay.fill_alpha = 0.2
+
+    select.line('date', 'points', source=real_source, line_color='blue')
+    select.line('date', 'points', source=pred_source, line_color='red')
+    select.ygrid.grid_line_color = None
+    select.add_tools(range_tool)
+    select.toolbar.active_multi = range_tool
+    select.add_layout(test_start)
+
+    chart = column(player_line, select)
+
+    st.bokeh_chart(chart)
+    st.write("{}'s dataframe:".format(player_name))
+    st.dataframe(series_dataframe)
+    return chart
+
+## HERE YE BE DRAGONS
```

Comment:
delete/comment out?

Reply:
What are best practices here? I don't want to remove the code as it was functional and I want to come back it; should it be moved into a new incomplete.py file or something along those lines?